src/c++/uds/userLinux/tests/Permassert_t1.c

```diff
@@ -99,15 +99,15 @@ static void checkNotFound(const char *str)
 /**********************************************************************/
 static void testAssertionSuccess(void)
 {
-  UDS_ASSERT_SUCCESS(ASSERT(true, "true"));
+  UDS_ASSERT_SUCCESS(VDO_ASSERT(true, "true"));
```

Comment:
I think this should be VDO_ASSERT_SUCCESS? Or are we only converting tests to just use VDO_ASSERT?

Reply:
This test is in the uds directory, so it doesn't have VDO_ASSERT_SUCCESS defined. As I mentioned in the commit message, I didn't think that was worth doing.